clients/python/test/e2e/ci/_utils.py

```diff
@@ -0,0 +1,17 @@
+from enum import IntEnum
+
+
+class E2eScriptFailure(UserWarning):
+    """Simply used to indicate a CI script failure"""
+
+    pass
+
+
+class E2eExitCodes(IntEnum):
+    """Exitcodes
+    Note these should not clash with pytest exitcodes: https://docs.pytest.org/en/7.1.x/reference/exit-codes.html
```

Comment:
TIP: use defensive programming and assert this condition programatically e.g.
```python
assert set(e.value for e in E2eExitCodes).intersection(e.value for e in pytest.ExitCode) == set()
```

Reply:
done